views/admin/users.erb

```diff
@@ -138,7 +189,49 @@
 
 <% append_script_declaration("WDN.initializePlugin('datepickers');") %>
 
+<% append_script_declaration("WDN.initializePlugin('pagination');") %>
+
 <% append_script_declaration(%q[require(['jquery'], function($) {
+
+    function buildSearchFilterList() {
```

Comment:
I was confused what the hidden values were for earlier but I see now. I'm not totally convinced that this is the best way to fix this bug, but I don't know a better way and I think that this is not an egregious solution. Might be good to think of other potential ways to solve this issue but for now I think its acceptable.

Reply:
This is the only way that I know how to get values from the server to the javascript without showing them to the users, but if we can come up with a better solution I can implement it along with any changes requested from the sponsors.